### DISCUSSION TITLE
How can I creat an Abimap?

### DISCUSSION THREAD

**Discussion by Gere321123** [Cyfrin/foundry-full-course-cu#2864] [Q&A]
Hi 
When I build my foundry project I get an out folder but I don't know how to use the abis I know how to create 1 correct abi like this :  <code>forge build --silent && jq '.abi' ./out/Coin.sol/Coin.json</code>

But I like to create 1 abimap witch have all my abis.
Because I use ERC20 and I think I also need this abi and for many more files.

Many thanks <3

**Comment by EngrPips** [comment]
I have never used this command `forge build --silent && jq '.abi' ./out/Coin.sol/Coin.json` lol Obviously I don't know what you are trying to achieve, but let's see what @PatrickAlphaC  have to say to you when they are available.

**Comment by LordAdhaar** [comment]
All ABIs should be located in the out folder as expected. If your ABI seems unreadable, try using a formatter like Prettier to improve readability.

![image](https://github.com/user-attachments/assets/8b4cb43b-7d35-4956-b50e-a49d5c28efc4)

Each entry in the abi array represents a function definition. If you need further help understanding how to use it, feel free to ask!
